src/main/webapp/app/exercises/programming/shared/code-editor/ace/code-editor-ace.component.ts

```diff
@@ -105,7 +121,21 @@ export class CodeEditorAceComponent implements AfterViewInit, OnChanges, OnDestr
     readonly faPlusSquare = faPlusSquare;
     readonly faCircleNotch = faCircleNotch;
 
-    constructor(private repositoryFileService: CodeEditorRepositoryFileService, private fileService: CodeEditorFileService, protected localStorageService: LocalStorageService) {}
+    /**
+     * Get all line numbers in the current editor session that have inline feedback or new feedback (which is only shown temporarily)
+     */
+    get linesWithInlineFeedbackShown(): number[] {
+        const lines = this.linesWithNewFeedback.concat(Object.keys(this.fileFeedbackPerLine).map((line) => parseInt(line)));
+        lines.sort((a, b) => a - b);
```

Comment:
Is the sorting relevant here?

Reply:
It's not strictly necessary, but it causes the HTML elements to be in the correct order, which helps debugging. Therefore, I will keep it.